### ISSUE TITLE
Training GPU Memory

### ISSUE THREAD

**Issue by OpenVE-Team** [Tencent-Hunyuan/HunyuanVideo-1.5#43]
Thank you for your excellent open-source work. I have a question about the training phase. When fine-tuning the HunyuanVideo-1.5-480P-I2V model with 8 GPUs (80GB VRAM each) and sp_size set to 8, I can only fine-tune 25 frames for a 480*832 resolution video. Any more frames result in an OOM. Is this normal?

**Comment by KimbingNg** [comment]
This OOM isn’t normal with gradient checkpointing and sp enabled. What is your training setting and batch_size, and can you provide your OOM error logs?
